COMMITTERS.adoc

```diff
@@ -0,0 +1,173 @@
+
+Obviously, each situation is different so use your own judgement,
+and if in doubt, just ask for advice, the other committers are here to help.
+
+== Pull Request Title
+
+The title should be in English, should not contain an issue number.
+It should also not contain ellipsis.
+
+If your commit message was too long and GitHub automatically cut
+the title, that's helpful if you can take the time to move the cut
+part where it belongs to have a full title.
+
+Titles are included in the Release notes so they are important.
+
+A good title would look like: `Fix XXX in Quartz extension` or
+`Introduce Hibernate Reactive extension`.
+
+A bad title would look like: `fix(#444)`.
+
+== Issues Fixed
+
+When a PR fixes some issues, it's good practice to add it in the description.
+
+One issue per line with something like:
+
+[source,asciidoc]
+----
+* Fix #444
+* Fix #555
+----
+
+[NOTE]
+====
+GitHub won't detect issues properly if you do something like
+`Fix #444 #555`.
+====
+
+== Affecting Labels and Milestones
+
+Affecting labels and milestones is very important in our process.
+
+Before each release, we check that we don't have any orphan i.e.
+closed issues that don't have either a milestone affected or
+some of the "excluding" labels.
+
+Thus:
+
+* when you merge a pull request, please affect it with the current milestone marked with "master".
+* if the pull request has issues associated (i.e. if the pull request fixes some issues),
+  you should also affect the milestone assigned to the issues.
+* if you close a pull request because the committers have decided to not merge it,
+  please add the appropriate `triage/` label: `triage/invalid`, `triage/out-of-date`,
+  `triage/wontfix` are usually in order.
+* if you close an issue because it has been fixed, either add the milestone
+  if you can find it easily or use the `triage/out-of-date` if you can't.
+* if you close an issue for any other reason, one of the aforementioned `triage/` labels
+  is probably adequate.
+
+== Backporting Process
+
+When we release a new version of Quarkus, we usually do a bugfix
+release a couple of weeks after.
+
+Everytime, we do a release, we create a release branch to host
+the commits for these bugfix releases.
+
+All the pull requests are merged in master so for the new feature 
+release of Quarkus.
+They won't be integrated in the previous version branch.
+
+One of your pull requests might qualify for being backported to this
+bugfix branch.
+
+Good examples of that might be:
+
+* bugfixes
+* documentation fixes
+* usability fixes
+
+Obviously, the barrier is higher for large pull requests as
+they might be more risky to backport.
+But sometimes, we just have to backport them anyway.
+
+If you think your pull request or the pull request you review might be a good backport candidate,
+please add the `triage/backport?` label.
+
+The question mark is important:
+it is not automatic and we carefully review each pull request before backporting.
```

Comment:
Perhaps also add a comment mentioning that it's possible to backport specific commits from a pull request if that pull request contains both fixes and new features

Reply:
Yeah I don't want to make that too prominent because it makes it a bit awkward for assigning milestones to the PR.

Reply:
OK :)